backend/OR-Tools/OR-Tools/ORSolver.cs

```diff
@@ -22,6 +22,7 @@ public void Solve()
         {
             // Create Routing Index Manager
             manager = new RoutingIndexManager(data.DistanceMatrix.GetLength(0), data.VehicleNumber, data.Starts, data.Ends);
+            //manager = new RoutingIndexManager(data.DistanceMatrix.GetLength(0), data.VehicleNumber, 0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4618867)